test/unit/application_test.rb

```diff
@@ -230,4 +230,25 @@ class ApplicationTest < ActiveSupport::TestCase
       end
     end
   end
+
+  describe "#latest_deploy_to_each_environment" do
+    should "orders environments" do
+      Deployment.delete_all
+
+      app = FactoryBot.create(:application, name: SecureRandom.hex, repo: "alphagov/#{SecureRandom.hex}")
+
+      production = FactoryBot.create(:deployment, application: app, environment: "production")
+      staging = FactoryBot.create(:deployment, application: app, environment: "staging")
+      integration = FactoryBot.create(:deployment, application: app, environment: "integration")
+
+      expected = {
+        "integration" => integration,
+        "staging" => staging,
+        "production" => production,
+      }
+
+      assert_equal(expected.keys, app.latest_deploy_to_each_environment.keys)
+    end
+  end
+
```

Comment:
The linter will probably complain about this unnecessary extra newline.

Reply:
Indeed it had, thank you, I have now fixed it. :)